Dockerfile

```diff
@@ -41,15 +41,14 @@ RUN rm -rf /var/lib/apt/lists/* && \
         | tar -xz -C /opt/ants-2.3.4 --strip-components 1
 
 RUN pip install -U pip
-RUN cd /tractolearn && pip install -e . 
+RUN cd /tractolearn && pip install -e .
```

Comment:
We are using here a bare `pip` whereas all other commands use the Python 3.8 version `pip`. I would use `pip` everywhere to save typing and make maintenance easier. As we are installing a single Python version, `pip` should be able to resolve to that verstion.

Reply:
Yep, I tested it on my side, you can remove the pip3.8

Reply:
Done in [268d578](https://github.com/scil-vital/dockerfile_tractolearn/pull/7/commits/268d578fa57f899c826f542f9055a68583682063).